chart/values.yaml

```
@@ -146,3 +146,32 @@ volumes:
   cache:
     emptyDir: {}
 
+# Enable this to use a custom network-config.json
+#customNetworkConfig: |
```

I think this should be just `config` and enabled by default. Also sort it within the file.
```suggestion
config: |
```